os.bpmsuite.businesscentral/added/launch/bpmsuite-businesscentral.sh

```diff
@@ -7,6 +7,12 @@ source $JBOSS_HOME/bin/launch/logging.sh
 
 function prepareEnv() {
     # please keep these in alphabetical order
+    unset APPFORMER_ELASTIC_CLUSTER_NAME
+    unset APPFORMER_ELASTIC_HOST
+    unset APPFORMER_ELASTIC_PORT
+    unset APPFORMER_ELASTIC_RETRIES
+    unset APPFORMER_JMS_BROKER_PASSWORD
+    unset APPFORMER_JMS_BROKER_USER
```

Comment:
Shouldn't be here also APPFORMER_JMS_BROKER_ADDRESS and APPFORMTER_JMS_BROKER_PORT ?